napari/utils/theme.py

```diff
@@ -239,10 +242,17 @@ def register_theme(name, theme):
     theme : dict of str: str, Theme
         Theme mapping elements to colors.
     """
-    if not isinstance(theme, Theme):
+    if isinstance(theme, dict):
         theme = Theme(**theme)
+    assert isinstance(theme, Theme)
     _themes[name] = theme
 
+    from qtpy.QtCore import QDir
+
+    from ..resources._icons import build_theme_svgs
+
+    QDir.addSearchPath(f'theme_{name}', build_theme_svgs(name))
```

Comment:
👀 ... this is the key line in this PR
https://doc.qt.io/qt-5/qdir.html#addSearchPath